app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_sep = command.split()
+    if command_sep[0] == "mv":
+        old_name = command_sep[1]
+        new_name = command_sep[-1].split("/")[-1]
+        directory = command_sep[-1].split("/")
+        way = ""
+        if len(directory) > 1:
+            way = "/".join(directory[0: -1: 1]) + "/"
```

Comment:
what if your program runs on another system where the path is delimited by `\` ?
use `os.path.join()` from module that u've imported in the first line.

Also think about how to improve your logic, 'cause this slicing is not the optimal solution